src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/UserViewModel.cs

```diff
@@ -16,11 +16,17 @@ public class UserViewModel
         
         public string Role { get; set; }
         
-        public List<UserReleaseRole> UserReleaseRoles { get; set; }
+        public List<UserReleaseRoleViewModel> UserReleaseRoles { get; set; }
     }
 
-    public class UserReleaseRole
+    public class UserReleaseRoleSubmission
```

Comment:
I'm not a big fan of this being in the same file and I would call it `Request` rather than `Submission` like we are using `Request` everywhere else.

Reply:
Ah cool, not a problem